USFMToolsSharp.Renderers.USX/USXConfig.cs

```diff
@@ -0,0 +1,32 @@
+namespace USFMToolsSharp.Renderers.USX
+{
+    public class USXConfig
```

Comment:
The only configuration that I foresee us needing is partial support everything else is stuff that we shouldn't need